Assets/Scripts/Models/Settings.cs

```diff
@@ -124,6 +138,14 @@ public static T GetSetting<T>(string key, T defaultValue)
 
     private static void SaveSettings()
     {
+        // if we do not have any unsaved settings then return
+        if (Settings.unsavedSettings == false)
+        {
+            Debug.ULogChannel("Settings", "No settings have changed, so none to save!");
```

Comment:
Now that its not a reoccurring event this log message makes more sense, as it would be unusual for the message to log.